Foundation/UserDefaults.swift

```diff
@@ -12,17 +12,85 @@ import CoreFoundation
 private var registeredDefaults = [String: Any]()
 private var sharedDefaults = UserDefaults()
 
+fileprivate func bridgeFromNSCFTypeIfNeeded(_ value: Any) -> Any {
+    // This line will produce a 'Conditional cast always succeeds' warning if compoiled on Darwin, since Darwin has bridging casts of any value to an object,
+    // but is required for non-Darwin to work correctly, since that platform _doesn't_ have bridging casts of that kind for now.
+    if let object = value as? AnyObject {
+        return _SwiftValue.fetch(nonOptional: object)
+    } else {
+        return value
+    }
+}
+
 open class UserDefaults: NSObject {
+    static private func _isValueAllowed(_ nonbridgedValue: Any) -> Bool {
+        let value = bridgeFromNSCFTypeIfNeeded(nonbridgedValue)
+        
+        if let value = value as? [Any] {
+            for innerValue in value {
```

Comment:
The swift tinkerer in me wants to say .lazy.filter { _isValueAllowed($0) }.count > 0, but tbh I’m not sure it’s worth it :)

Reply:
I will reply with my standard reply, which is 'if it turns out to be a hot spot'.